hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/volume/OMVolumeSetQuotaResponse.java

```diff
@@ -29,29 +29,36 @@
 import java.io.IOException;
 
 import javax.annotation.Nonnull;
+import org.apache.ratis.util.Preconditions;
 
 /**
  * Response for set quota request.
  */
 public class OMVolumeSetQuotaResponse extends OMClientResponse {
   private OmVolumeArgs omVolumeArgs;
 
-  public OMVolumeSetQuotaResponse(OmVolumeArgs omVolumeArgs,
-      @Nonnull OMResponse omResponse) {
+  public OMVolumeSetQuotaResponse(@Nonnull OMResponse omResponse,
+      OmVolumeArgs omVolumeArgs) {
```

Comment:
Can we add @NonNull check for other params too, as now we have another constructor for error/replay scenario. The same comment applies to all classes.